config/services.yaml

```diff
@@ -1,3 +1,6 @@
+parameters:
+    locale: 'en'
```

Comment:
how do people configure this (if it needs to be configured)? Should it have an environment variable that can be used?

Reply:
(used to provide this value to the container)